+bisos/+iteration/+step/termination.m

```diff
@@ -0,0 +1,134 @@
+        case {'<', 'lt'}
+           step.operator = @lt; 
+        case {'>', 'gt'}
+           step.operator = @gt;  
+        case {'==', 'eq'}
+           step.operator = @eq;  
+        case {op, '~=', 'ne'}
+           step.operator = @ne;
+        otherwise
+           error('Operator "%s" not valid.', op)
+        end
+    end
+    
+    function [sol,info,stop] = run(step,~,info,sol,varargin)
+        % Run objective step.
+        stop = false;
+
+        if info.iter > 1
+            stepinfo = getinfo(step,info);
+        
+            % Prepare current term for condition evaluation    
+            value2 = step.calcfhan(sol, step.curr);
+        
+            % Get previous term for condition evaluation
+            value1 = stepinfo.value;
+
+            if step.operator(value1, value2)
+                info.converged = true;
+            end
+        end
+        
+        % Store 'previous' term for next iteration
+        stepinfo.value = step.calcfhan(sol, step.prev);
+        
+        info = setinfo(step,info,stepinfo);
+    end
+end
+
+methods (Static, Access=private)
+    function value = calcfhan(sol, sstruct) 
+        % sol: solutions that are accessed 
+        % sstruct: state struture, it can be either previous or current
+        % state
+
+        % preallocate auxiliary variable 'v'
+        v = cell(1,length(sstruct.vars));
+
+        for i=1:length(sstruct.vars)
+            if isfield(sol, sstruct.vars{i})
+                v{i} = double(sol.(sstruct.vars{i}));
+            else
+                error('Non existance of variable in solutions.');
+            end
+        end
+        value = sstruct.fhan(v{:});
+    end
+
+    function  defstruct = setup2struct(defstruct, cdef)
+
+        if length(cdef)>1 
+            if isa(cdef{1}, "function_handle")
+                defstruct.fhan = cdef{1};
+                
+            % Check coherence between the number of inputs defined in 
+            % functions handle and the number of inputs given
+                if nargin(defstruct.fhan) ~= (length(cdef)-1)
+                    error('Inconsistency between function handle and number of input variables.');
+                end
+                
+                % save variables name in defstruct
+                for i=2:length(cdef)
+                    % verify if input data type os 'char'
+                    if ~ischar(cdef{i})
+                        error('error in variables type.');
+                    else
+                        defstruct.vars{end+1} = cdef{i};
+                    end
+                end
+            else
+                error('No function handle is given.')
+            end
+        elseif ~isempty(cdef)
+
+            if ~ischar(cdef{:})
+                error('error in variable type.');
+            else
+                defstruct.vars = cdef;
+            end
+        % in case no input is given
+        else
+            error('Some input variable is empty.')
```

Comment:
as [commented above](https://github.com/tcunis/bisosprob/pull/13/files#r908913730), just do in the beginning

```
assert(~isempty(cdef), 'No input variables given.')